contrib/babelfishpg_tsql/src/pltsql_function_probin_handler.c

```diff
@@ -62,14 +62,20 @@ pltsql_function_probin_writer(CreateFunctionStmt *stmt, Oid languageOid, char **
 	char	   *langname;
 	int			probin_len;
 	Jsonb	   *jb;
+	bool		dump_restore = false;
+	const char *babelfish_dump_restore = GetConfigOption("babelfishpg_tsql.dump_restore", true, false);
```

Comment:
Why do we need additional handling during dump/restore? IIUC we only need to handle it in upgrade script.

Reply:
I can see that changes in SQL upgrade script are sufficient for almost all MVU/mVU paths. The failures are only in some upgrade path which involves 14.6/13.6 version. So I would suggest to narrow down the issue, instead of making changes in dump/restore.

Reply:
As the SQL upgrade script was working for all the upgrade paths from 14.6 and ahead. After discussion with Rishabh, we narrowed down the issue, and found that in the upgrade query we are using `sys.babelfish_function_ext` to filter out the bbf defined functions. But this catalog was introduced in 14.6, so the following upgrade paths were failing as the procedure information was not stored for them in the `sys.babelfish_function_ext` catalog and hence was not getting updated:
13_6-14_6-target_latest
13_9-14_6-target_latest
14_3-14_6_target-latest
14_5-14_6-target_latest
So we concluded to use pg_language catalog and using the pltsql filter to narrow down the bbf defined procedures and functions.